server/src/main/java/org/opensearch/cluster/node/DiscoveryNode.java

```diff
@@ -476,6 +478,16 @@ public boolean isRemoteStoreNode() {
         return this.getAttributes().keySet().stream().anyMatch(key -> key.startsWith(REMOTE_STORE_NODE_ATTRIBUTE_KEY_PREFIX));
     }
 
+    public boolean isRemoteIndexStoreEnabled() {
```

Comment:
[minor]: just isRemoteStoreEnabled to keep it consistent

Reply:
There is already method `isRemoteStoreEnabled ` which looks up the attribute based on the prefix `remote_store`. Does `isIndexRemoteStoreEnabled` sound better ? (in-line with setting index.remote_store.enabled)

Reply:
added the check to `isRemoteStoreEnabled`